pkg/http/api.go

```diff
@@ -317,6 +317,12 @@ func (a *api) constructHealthzEndpoints() []Endpoint {
 			Version: apiVersionV1,
 			Handler: a.onGetHealthz,
 		},
+		{
+			Methods: []string{fasthttp.MethodGet},
+			Route:   "healthz/{storeName}",
```

Comment:
This needs to be `healthz/state/{storeName}` to allow for other components to have health checks.

In addition, we probably need a gRPC equivalent for this.

Reply:
@yaron2  I have added changes about gRPC and URI. PTAL.